sidebars.js

```diff
@@ -42,6 +42,7 @@ module.exports = {
         'guides/syndication',
         'guides/sca',
         'guides/page-protection',
+        'guides/web-payments',
```

Comment:
```suggestion
        'guides/enable-web-payments-with-stripe',
```

The id of the page is set to `enable-web-payments-with-stripe`. 
The page is already published here: https://developers.inplayer.com/docs/enable-web-payments-with-stripe/